app/transformers/rural-payments/business.js

```diff
@@ -111,11 +111,16 @@ export const transformOrganisationToBusiness = (data) => ({
   sbi: `${data?.sbi}`
 })
 
+const convertValidatedAddressToKitsAddress = (address) => {
+  const addr = address?.withUprn || address?.withoutUprn
+  return addr ? dalAddressToKitsAddress(addr) : undefined
+}
+
 const orgDetailsUpdateMapping = {
   name: (data) => data.name,
-  address: (data) => (data.address ? dalAddressToKitsAddress(data.address) : undefined),
+  address: (data) => convertValidatedAddressToKitsAddress(data?.address),
```

Comment:
Can' you just do:
```suggestion
  address: ({ address }) => address && dalAddressToKitsAddress(address?.withUprn || address.withoutUprn),
```
...then you don't need the extra function - or...